src/bakabot/main.py

```diff
@@ -55,15 +61,19 @@ async def on_raw_reaction_add(reaction: disnake.RawReactionActionEvent):
 
 
 # Starts couroutines needed for some features
-async def start_feature_couroutines(client: disnake.Client):
-    if await first_time_setup.start(client):
-        # Starts the courutines
-        await asyncio.gather(
-            Reactions.query(client),
-            ChangeDetector.start_detecting_changes(60, client),
-            Grades.start_detecting_changes(60, client),
-            Reminder.start_reminding(client),
-        )
+async def start_feature_couroutines(client: InteractionBot):
+    await asyncio.gather(
+        Reactions.query(client),
+        ChangeDetector.start_detecting_changes(60, client),
+        Grades.start_detecting_changes(60, client),
+        Reminder.start_reminding(client),
+    )
+
+
+def initialize_features():
+    """Initializes key components of the bot"""
+
+    SubjectsCache.initialize()
```

Comment:
The subjects cache is initialized each time the bot is started, as afterward all the data is in memory, it's only saved to disc